client/electron/go_plugin.ts

```diff
@@ -36,7 +34,105 @@ export async function invokeGoMethod(
   method: string,
   input: string
 ): Promise<string> {
-  if (!invokeMethodFunc) {
+  console.debug(`[Backend] - calling InvokeMethod "${method}" ...`);
+  const result = await ensureCgo().invokeMethod(method, input);
+  console.debug(`[Backend] - InvokeMethod "${method}" returned`, result);
+  if (result.ErrorJson) {
+    throw new Error(result.ErrorJson);
+  }
+  return result.Output;
+}
+
+/**
+ * Represents a function that will be called from the Go backend.
+ * @param data The data string passed from the Go backend.
+ */
+export type CallbackFunction = (data: string) => void;
+
+/** A unique token for a callback created by `newCallback`. */
+export type CallbackToken = string;
```

Comment:
I suggest we stick with just a consistent callback id throughout the stack. No need to have both an id and a token.

Reply:
Done.